docs/UserGuide.adoc

```diff
@@ -509,6 +509,8 @@ This command will remove information in the storage.
 
 You do not need to manually save the data.
 
+However, user should take note not to modify the default files in anyway. Manipulating the data inside files such as Medeina.xml, might cause exceptions (the application may still run, but will not be able to return error messages).
```

Comment:
I think u can put a warning here.

Reply:
I'll keep the header as Data Saving, and add a warning at the end to prevent user from modifying the files. Is that okay with you?